<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scenExecution</class>
 <widget class="QDialog" name="scenExecution">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="exec_label1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Scenario...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stop_Button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>stop</string>
   </property>
  </widget>
  <widget class="QLabel" name="progress_label1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Progress:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openLogFile_Button">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>open log file</string>
   </property>
  </widget>
  <widget class="QLabel" name="progress_label2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
